pkgs/development/compilers/ghc/8.10.1.nix

```diff
@@ -84,6 +84,8 @@ let
 
   targetCC = builtins.head toolsForTarget;
 
+  # ld.gold is disabled for musl libc due to https://sourceware.org/bugzilla/show_bug.cgi?id=23856
+  # see #84670 and #49071 for more background.
   useLdGold = targetPlatform.isLinux && !(targetPlatform.useLLVM or false) && !targetPlatform.isMusl;
```

Comment:
`!(targetPlatform.useLLVM or false)` is semantically identical to `!targetPlatform.useLLVM`, maybe you could simplify these?

Reply:
Done

Reply:
It seems like this change causes the evaluation to fail with `attribute 'useLLVM' missing`. This is probably the reason for the `or false`.

https://gist.github.com/GrahamcOfBorg/8ba5adb98b2fc7b35700fa17dff9150a

Reply:
Whoops, sorry for the noise; I had no idea about the `or` operator.

Though to be fair, I doubt anyone else does either, since it's documented only in [release notes from 2012](https://nixos.org/nix/manual/#ssec-relnotes-1.0)...